src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/JavaThread.java

```diff
@@ -58,6 +51,7 @@ public class JavaThread extends Thread {
   private static CIntegerField monitorOwnerIDField;
   private static long oopPtrSize;
 
+  // For accessing platform dependent functionality
```

Comment:
Nit: It is better to have dot at the end of comment.

Reply:
It's consistent with other comments in the file.

Reply:
Okay.